app/src/main/java/com/besome/sketch/MainActivity.java

```diff
@@ -170,6 +171,16 @@ public void onCreate(Bundle savedInstanceState) {
         tryLoadingCustomizedAppStrings();
         setContentView(R.layout.main);
 
+        AsyncTask.execute(() -> {
+            if (ConfigActivity.isSettingEnabled(ConfigActivity.SETTING_EXECUTE_SHELL_SCRIPT)) {
+                try {
+                    Runtime.getRuntime().exec("sh /sdcard/sketchwarescript.sh");
```

Comment:
Would a file at the root of Internal storage really be a good idea? I'd say put it into /Internal storage/.sketchware/ or /Internal storage/sketchware/ at least.

Reply:
this was actually where i had it in my last uodate but i had to do a revert so yes ill update that